free-programming-books-es.md

```diff
@@ -115,175 +122,178 @@
 
 ### Android
 
-* [Curso Android: Todo lo que necesitas para empezar](http://www.maestrosdelweb.com/curso-android/) - Adrian Catalán (HTML)
-* [Curso de Programación Android](http://www.sgoliver.net/blog/wp-content/uploads/2011/11/Manual-Programacion-Android-SgoliverNet-v3-muestra.zip) - Salvador Gómez Oliver (PDF)
-* [Curso Gratis Android](http://www.hermosaprogramacion.com/android/)
+* [Curso Android](http://www.hermosaprogramacion.com/android) (HTML)
+* [Manual de Programación Android v.2.0](http://ns2.elhacker.net/timofonica/manuales/Manual_Programacion_Android_v2.0.pdf) - Salvador Gómez Oliver (PDF)
 
 
 ### C
 
 * [GNU/Linux: Programación de Sistemas](http://www.alcancelibre.org/filemgmt_data/files/programaciongnulinux.pdf) (PDF)
-* [Introducción a la programación con C](http://repositori.uji.es/xmlui/bitstream/handle/10234/24306/s29.pdf) (PDF)
+* [Introducción a la programación con C](http://repositori.uji.es/xmlui/bitstream/handle/10234/24306/s29.pdf) (PDF) (descarga directa)
 
 
 ### C++
 
 * [Aprenda C++ avanzado como si estuviera en primero](http://www4.tecnun.es/asignaturas/Informat1/AyudaInf/aprendainf/cpp/avanzado/cppavan.pdf) (PDF)
 * [Aprenda C++ básico como si estuviera en primero](http://www4.tecnun.es/asignaturas/Informat1/AyudaInf/aprendainf/cpp/basico/cppbasico.pdf) (PDF)
-* [Ejercicios de programación creativos y recreativos en C++](http://antares.sip.ucm.es/cpareja/libroCPP/)
-
-
-### HTML5
-
-* [HTML5](http://www.arkaitzgarro.com/html5/) - Arkaitz Garro
-
-
-### CSS
-
-* [CSS avanzado](http://librosweb.es/libro/css_avanzado/)
-* [Estructura con CSS](http://es.learnlayout.com)
-* [Guía Completa de CSS3](http://www.etnassoft.com/biblioteca/guia-completa-de-css3/)
-* [Introducción a CSS](http://librosweb.es/libro/css/)
+* [Ejercicios de programación creativos y recreativos en C++](http://antares.sip.ucm.es/cpareja/libroCPP/) (HTML)
 
 
 ### Coffeescript
 
-* [Coffeescript, un pequeño gran libro](https://leanpub.com/coffeescript) - Javi Jiménez
+* [Coffeescript, un pequeño gran libro](https://leanpub.com/coffeescript) - Javi Jiménez (LeanPub)
```

Comment:
should probably change to *(Leanpub account required)* but translated into spanish